tmtools/tests/test_io.py

```diff
@@ -3,19 +3,23 @@
 import numpy.testing as nptest
 
 from ..io import get_residue_data, get_structure
-from ..testing import get_pdb_path
+from ..testing import get_pdb_path, get_mmcif_path
 
 
 class TestIO(unittest.TestCase):
     def test_get_structure(self):
-        # Given
```

Comment:
Could you maintain the given/when/then structure here? If those terms are too generic, you can add some explanation, e.g. `# Given - a valid PDB file`, etc.

Also, don't worry about creating a separate test for mmcif, even if it duplicates some code from the pdb test. Likewise if the test ends up checking many things at the same time, that is usually a sign that it should be broken out into multiple tests. Here I would maybe suggest 3 tests: (a) loading PDB, (b) loading MMCIF, (c) checking that the structures are equal.

Reply:
No problem! Please check the changes I did, basically the logic was the following:
- **Given** an structure ID
- **When** we load the structure from PDB/MMCIF
- **Then** we compare both structures to see if this are equal